packages/mail/integration.test.js

```diff
@@ -0,0 +1,132 @@
+require('dotenv').config({ path: '../../.test.env' })
```

Comment:
IMHO we should move this file to republik-backend to completly keep this "republik-free".